me.id.webverify/.gitignore

```diff
@@ -1,6 +1,5 @@
 .gradle
 /local.properties
-/.idea/workspace.xml
-/.idea/libraries
+/.idea
```

Comment:
yes, I know, but I want to ignore only the root project .idea folder. What do you think? should we ignore all .idea folders?